### PR TITLE
Update build to latest infrastructure

### LINKED ISSUE #92: Update build
Update the build to use the new infrastructure

### PR DESCRIPTION
**Pull request by timyates**
Keeping micronaut version the same.  I will raise another PR to update micronaut to the latest 3.3.x
and fix the missing default for secure cookies

Fixes #92

### CODE REVIEW

**Comment by timyates** [comment]
Actually, it may not be a pressing concern to update to micronaut 3.  The only change is requiring rxjava2 for the tests @sdelamo WDYT?

**Comment by timyates** [comment]
Pushed the javadoc fix here as well

File: settings.gradle [hunk 1]
```diff
@@ -1,3 +1,14 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("io.micronaut.build.shared.settings") version "5.2.1"
```
Comment (sdelamo) [inline settings.gradle:9]: this should be 5.2.2

File: gradle.properties [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
-micronautDocsVersion=2.0.0.RC1
+projectGroup=io.micronaut.multitenancy
+
+micronautDocsVersion=2.0.0
 micronautVersion=2.5.12
-micronautTestVersion=2.3.8
-groovyVersion=3.0.8
+micronautTestVersion=2.3.7
+
+groovyVersion=3.0.9
```
Comment (sdelamo) [inline gradle.properties:8]: use 3.0.10

**Review by sdelamo — CHANGES_REQUESTED** [review]

**Pull request merged**